tests/certification/state/mongodb/README.md

```diff
@@ -0,0 +1,24 @@
+# MongoDB State Store certification testing
+
+This project aims to test the MongoDB State Store component under various conditions.
+
+## Test plan
+
+## Basic Test for CRUD operations:
+1. Able to create and test connection.
+2. Able to do set, fetch, update and delete.
+3. Negative test to fetch record with key, that is not present.
+
+## Component must reconnect when server or network errors are encountered
+
+## Infra test:
+1. When MongoDB goes down and then comes back up - client is able to connect
+
+## eTag related:
+1. Insert a Key-Value pair, eTag will be 1  
+2. Update Value v2 for this Key with eTag equal to 1 - new eTag wil be 2.  
+3. Try to Update v3 for this Key with eTag equal to 4 - value should not get updated.  
+4. Confirm that the Value is v2 for this Key.  
+
+## Transaction related, like Upsert:
```

Comment:
This is also already covered by conformance tests.
https://github.com/dapr/components-contrib/blob/master/tests/config/state/tests.yml#L6

Reply:
Fixed